jina/orchestrate/pods/__init__.py

```diff
@@ -340,6 +339,7 @@ def __init__(self, args: 'argparse.Namespace'):
                 else None,
                 'runtime_cls': self.runtime_cls,
                 'jaml_classes': JAML.registered_classes(),
+                'daemon': True,
```

Comment:
There is some `_get_worker` or `process` in every kind of `Pod`, needs to be done also for `ContainerPod` at least